docs/mp/integration/wls.adoc

```diff
@@ -0,0 +1,669 @@
+
+=== Example [[JMS-Integration-Examples]]
+
+The following example shows the helidon-jms connector configurations added to the `application.yaml` file.
+
+[source,java]
+----
+# User-defined properties
+wls-username: weblogic
+wls-password: welcome1
+# WLS Admin server t3 connection URL
+wls-admin-url: t3://localhost:7001
+# WLS Cluster t3 connection URL
+wls-cluster-url: t3://localhost:7003,localhost:7005,localhost:7007
+ 
+mp:
+  messaging:
+    connector:
+      helidon-jms:
+        jndi:
+           #Default connection factory name. This can be overridden in individual resource configurations
+           jms-factory: qcf
+           #JMS environment properties to lookup resources
+           env-properties:
+              java.naming.factory.initial: weblogic.jndi.WLInitialContextFactory
+              java.naming.provider.url: ${wls-admin-url}
+              java.naming.security.principal: ${wls-username}
+              java.naming.security.credentials: ${wls-password}
+ 
+    # Add all consumer resources-related configurations below incoming 
+    incoming:
+      #Identifier "from-wls-q" is used with @Incoming annotation
+      from-wls-q:
+        #Connector Name as specific in connector section. It is predefined
+        connector: helidon-jms
+        #JMS Destination Name in JNDI format
+        jndi.destination: queuejndi
+        #JMS Destination Type
+        type: queue
+        #JMS Connection Factory 
+        jndi.jms-factory: qcf
+ 
+    # Add all producer resources-related configurations below outgoing 
+    outgoing:
+      #Identifier "to-wls-q" is used with @Outgoing annotation
+      to-wls-q:
+        connector: helidon-jms
+        jndi.destination: queuejndi
+        type: queue
+        jndi.jms-factory: qcf
+----
+
+=== References [[JMS-Integration-References]]
+
+* https://helidon.io/docs/v3/#/mp/reactivemessaging/jms[JMS Connector]
+https://docs-uat.us.oracle.com/en/middleware/standalone/weblogic-server/14.1.1.0/wlshe/oracle-weblogic-server-helidon-integration.html#GUID-F7E4D3C2-6FE3-4C6E-B69B-805667B78312[Integrating Oracle WebLogic Server with Helidon]
+
+== Web Services Integration [[Web-Services-Integration]]
+
+- <<Web-Services-Integration-Overview, Overview>>
+- <<Web-Services-Integration-Maven-Coordinates, Maven Coordinates>>
+- <<Web-Services-Integration-Usage, Usage>>
+- <<Web-Services-Integration-Example, Example>>
+- <<Web-Services-Integration-References, References>>
+
+=== Overview [[Web-Services-Integration-Overview]]
+
+Helion MP and WebLogic Server Web Services integration enables the Helidon microservice application to communicate with the WebLogic Web Service deployed in Oracle WebLogic Server.
+
+=== Maven Coordinates [[Web-Services-Integration-Maven-Coordinates]]
+
+Add the `com.oracle.webservices.wls.jaxws-wlswss-client.jar` client file downloaded from https://edelivery.oracle.com/osdc/faces/Home.jspx[Oracle Software Delivery Cloud], as part of the Maven dependencies:
```

Comment:
this client should not be downloaded from edelivery. The  com.oracle.webservices.wls.jaxws-wlswss-client.jar is generated when WebLogic Server is installed and It is located in the WL_HOME\server\lib directory of the WebLogic Server installation.